sensor_msgs/Image image
float64 steering_angle
float64 lane_offset
builtin_interfaces/Time camera_time